research/picai/monai_scripts/auto3dseg.py

```diff
@@ -0,0 +1,153 @@
+"""Uses Auto3dSeg from monai to train a segmentation model
+
+-Assumes dataset is formatted following nnUNet structure for consistency
+"""
+
+import argparse
+import os
+from os.path import join
+from pathlib import Path
+from typing import Dict, Optional
+
+import numpy as np
+import yaml
+from batchgenerators.utilities.file_and_folder_operations import save_json
+from monai.apps.auto3dseg.auto_runner import AutoRunner
+
+
+def gen_dataset_list(data_dir: Path, output_path: Optional[str] = None, ext: str = ".nii.gz") -> str:
+    """
+    Generates a MONAI dataset list for an nnUNet structured dataset
+
+    Note: Rather than having a single image and label, this checks for multiple \
```

Comment:
Great documentation! One small nit is that the comments in triple quotes don't require the \ at the end